krakenapi.go

```diff
@@ -580,7 +580,7 @@ func (api *KrakenAPI) Query(method string, data map[string]string) (interface{},
 // Execute a public method query
 func (api *KrakenAPI) queryPublic(method string, values url.Values, typ interface{}) (interface{}, error) {
 	url := fmt.Sprintf("%s/%s/public/%s", APIURL, APIVersion, method)
-	resp, err := api.doRequest(url, values, nil, typ)
+	resp, err := api.doRequest(http.MethodGet, url, values, nil, typ)
```

Comment:
Is there a reason not to use `method` param passed into this function here as well?

Reply:
All public Kraken methods are `GET` [according to the docs](https://docs.kraken.com/rest/#tag/Market-Data) so we can just always use `http.MethodGet`. Could add another arg for future proofing though.